CMakeLists.txt

```diff
@@ -6,8 +6,17 @@ project(everest-timer
     LANGUAGES CXX C
 )
 
-find_package(everest-cmake 0.1 REQUIRED
-    PATHS ../everest-cmake
```

Comment:
The changea in this file ahould be behind a 
`if (USE_FIND_PACKAGE)`
block like in libocpp, since we still want to use find_package in some cases not exclusively fetchcontent

Reply:
Added that, can you take a look if it is ok right now?